[2024-03-18T06:46:00.127+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: two_separate_groups.bigquery_external_table_1_task manual__2024-03-18T06:40:31.356042+00:00 [queued]>
[2024-03-18T06:46:00.172+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: two_separate_groups.bigquery_external_table_1_task manual__2024-03-18T06:40:31.356042+00:00 [queued]>
[2024-03-18T06:46:00.179+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-03-18T06:46:00.239+0000] {taskinstance.py:2214} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_1_task> on 2024-03-18 06:40:31.356042+00:00
[2024-03-18T06:46:00.247+0000] {standard_task_runner.py:60} INFO - Started process 512 to run task
[2024-03-18T06:46:00.282+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'two_separate_groups', 'bigquery_external_table_1_task', 'manual__2024-03-18T06:40:31.356042+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/ingest_data_to_gcs.py', '--cfg-path', '/tmp/tmpjoyoa3p0']
[2024-03-18T06:46:00.294+0000] {standard_task_runner.py:88} INFO - Job 257: Subtask bigquery_external_table_1_task
[2024-03-18T06:46:00.555+0000] {task_command.py:423} INFO - Running <TaskInstance: two_separate_groups.bigquery_external_table_1_task manual__2024-03-18T06:40:31.356042+00:00 [running]> on host 9958f290dc4a
[2024-03-18T06:46:01.081+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='two_separate_groups' AIRFLOW_CTX_TASK_ID='bigquery_external_table_1_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T06:40:31.356042+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T06:40:31.356042+00:00'
[2024-03-18T06:46:01.090+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-03-18T06:46:01.093+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-18T06:46:01.094+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-18T06:46:02.261+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=two_separate_groups, task_id=bigquery_external_table_1_task, execution_date=20240318T064031, start_date=20240318T064600, end_date=20240318T064602
[2024-03-18T06:46:02.390+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T06:46:02.458+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
